src/character-sheet/BasicInformation.js

```diff
@@ -0,0 +1,298 @@
+                                        overflow: "hidden",
+                                        textOverflow: "ellipsis",
+                                    }}
+                                    onChange={handleChange}
+                                >
+                                    {CLASSES.map((story) => {
+                                        return (
+                                            <MenuItem
+                                                // classes={{ root: classes.root }}
+                                                key={story}
+                                                value={story}
+                                                divider
+                                            >
+                                                {story}
+                                            </MenuItem>
+                                        );
+                                    })}
+                                </Select>
+                                <FormHelperText>Class</FormHelperText>
+                            </FormControl>
+                        </Item>
+                    </Grid>
+                    <Grid item xs={3}>
+                        <Item>
+                            <Typography name="level" sx={{ fontSize: 40 }}>
+                                {props.character.level}
+                            </Typography>
+                            <FormHelperText>Level</FormHelperText>
+                        </Item>
+                    </Grid>
+                    <Grid item xs={2}>
+                        <Grid container direction="column">
+                            <Grid item xs={12}>
+                                <Item>
+                                    <Button
+                                        onClick={handleLevelButton}
+                                        name="lvlUp"
+                                        disabled={
+                                            props.character.selectedClass ===
+                                                "" ||
+                                            props.character.origin === ""
+                                        }
+                                    >
+                                        LV. Up
+                                    </Button>
+                                </Item>
+                            </Grid>
+                            <Grid item xs={12}>
+                                <Item>
+                                    <Button
+                                        onClick={handleLevelButton}
+                                        name="lvlDown"
+                                        disabled={
+                                            props.character.selectedClass ===
+                                                "" ||
+                                            props.character.origin === ""
+                                        }
```

Comment:
instead of doing triple = sign you could do !props.character.selectedClass. 
https://stackoverflow.com/questions/6003884/how-do-i-check-for-null-values-in-javascript